### DISCUSSION TITLE
I cant connect my feed in RSSHUB

### DISCUSSION THREAD

**Discussion by ghost** [DIYgod/RSSHub#15767] [Q&A]
Hi, 
I can't configure my feed in my rsshub for the https://www.proofpoint.com/us/blog/cloud-security site in the cloud security section. 

![image](https://github.com/DIYgod/RSSHub/assets/91891806/22ec6549-a8e9-45f5-8218-6c454b7c55a8)

![image](https://github.com/DIYgod/RSSHub/assets/91891806/c4e5a23f-d18d-4d9a-b17b-e15ec63ee76e)

![image](https://github.com/DIYgod/RSSHub/assets/91891806/5e345a22-2df0-4038-8522-ba933e995383)

**Comment by TonyRL** [comment]
I suppose you didn't notice the bolded words in your second picture.
